PTMagic/Program.cs

```diff
@@ -26,7 +26,7 @@ static void Main(string[] args)
       // Keep the app running
       for (; ; )
       {
-        Thread.Sleep(100);
+        Thread.Sleep(10000);
```

Comment:
Why change that value?

Reply:
This is essentially causing a thread interrupt 10 times a second, so seemed a bit wasteful as we are just blocking the main thread from exiting.

Reply:
Yeah makes sense